code/practice/4.py

```
@@ -0,0 +1,34 @@
+class UnderAgeError(Exception):
+    pass
+
+try:
+    age = int(input("年齢を入力してください: "))
+    if age < 18:
+        raise UnderAgeError("18歳未満は利用できません")
+except UnderAgeError as e:
+    print(e)
+except ValueError:
+    print("数字を入力してください")
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)